docs/quick-ingestion-guides/tableau/configuration.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Configuration
+---
+# Configuring Your Tableau Connector to DataHub
+
+Now that you have created a DataHub-specific user with the relevant access in Tableau in [the prior step](setup.md), it's now time to set up a connection via the DataHub UI.
+
+## Configure Secrets
+
+1. Within DataHub, navigate to the **Ingestion** tab in the top, right corner of your screen
+
+<p align="center">
+  <img width="75%" alt="Navigate to the &quot;Ingestion Tab&quot;" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_ingestion_button.png"/>
+</p>
+
+:::note
+If you do not see the Ingestion tab, please contact your DataHub admin to grant you the correct permissions
+:::
+
+2. Navigate to the **Secrets** tab and click **Create new secret**
+
+<p align="center">
+   <img width="75%" alt="Secrets Tab" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_secrets_tab.png"/>
+</p>
+
+3. Create a `username` secret
+
+  This will securely store your Tableau `username` within DataHub
+
+   * Enter a name like `TABLEAU_USERNAME` - we will use this later to refer in recipe
+   * Enter the `username`, setup in the [setup guide](setup.md)
+   * Optionally add a description
+   * Click **Create**
+
+  <p align="center">
+    <img width="70%" alt="Tableau Username Secret" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/tableau/tableau-username-secret.png"/>
+  </p>
+
+4. Create a `password` secret
+
+  This will securely store your Tableau `password` within DataHub
+
+   * Enter a name like `TABLEAU_PASSWORD` - we will use this later to refer in recipe
+   * Enter the `password` of the user, setup in the [setup guide](setup.md)
+   * Optionally add a description
+   * Click **Create**
+
+  <p align="center">
+    <img width="70%" alt="Tableau Password Secret" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/tableau/tableau-user-password-secret.png"/>
+  </p>
+## Configure Recipe
+
+5. Navigate to the **Sources** tab and click **Create new source**
+
+<p align="center">
+  <img width="75%" alt="Click &quot;Create new source&quot;" src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/guides/common/common_ingestion_click_create_new_source_button.png"/>
+</p>
+
+6. Select Tableau
+
+<p align="center">
+  <img width="70%" alt="Select Tableau from the options" src="https://raw.githubusercontent.com/datahub-project/static-assets-fork/main/imgs/guides/tableau/tableau-new-ingestion-source.png"/>
+</p>
+
+7.  Enter details into the Tableau Recipe
```

Comment:
Since it's in a separate section should be new list.

Here is an idea about how to make it more concise:


5. Navigate to on the **Sources** tab and then **Create new source**.

6. Select Tableau.

7. Fill in the Tableau Recipe form:

a. **Host URL**: URL of your Tableau instance (e.g., https://15az.online.tableau.com/).

b. **Username**: Use the TABLEAU_USERNAME secret (e.g., "${TABLEAU_USERNAME}").

c. **Password**: Use the TABLEAU_PASSWORD secret (e.g., "${TABLEAU_PASSWORD}").

To filter for specific site and project, use the site and project_pattern fields. Click **Next** when done.